syntax = "proto3";

package echo;

message EchoRequest {
    string message = 1;
    int64 timestamp = 2;
}

message EchoResponse {
    string message = 1;
    int64 timestamp = 2;
    string server_id = 3;
}
